R/kernel.r

```diff
@@ -198,9 +198,11 @@ main <- function(connection_file="") {
 
 #'Install the kernelspec to tell IPython (>= 3) about IRkernel
 #'
+#'@param user Install into user directory ~/.ipython or globally?
 #'@export
-installspec <- function() {
+installspec <- function(user=F) {
```

Comment:
Let's make the default True - since R packages typically install to a per-user directory, it makes sense for the kernelspec to follow suit. And that way, there's no need to update the README.